app/javascript/App/bundles/CurrencyWidget/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import CurrencySelector from '~/bundles/common/components/CurrencySelector'
+import CurrencyContext, {
+  CurrencyContextType,
+} from '~/bundles/common/contexts/CurrencyContext'
+
+const CurrencyWidget = () => (
```

Comment:
What do you think about calling this `CurrencySelectorWidget` so its a little more descriptive about what this component does